controllers.js

```diff
@@ -67,10 +68,27 @@ const postCommentsByArticleId = (req, res, next) => {
     });
 };
 
+const patchArticleByArticleId = (req, res, next) => {
+  const { inc_votes } = req.body;
+  const article_id = req.params.article_id;
+
+  fetchArticleById(article_id)
+    .then(() => {
+      return updateArticleByArticleId(article_id, inc_votes);
```

Comment:
You don't need to fetchArticlebyId before doing an update, just the update should suffice. If you are doing this to check that the article exists, you don't need to. If you entered an invalid article for update then psql will still throw an error, if you enter a valid but not found article then you can deal with that in the model if you get 0 rows back. 

With this method, and I can see why you've done it, I would say the biggest drawback is you are doing two queries to the database when just one will do!